docs/source/how_to/how_to_specify_algorithm_and_algo_options.md

```diff
@@ -1,46 +1,153 @@
 (specify-algorithm)=
 
-# How to specify algorithms and algorithm specific options
+# How to specify and configure algorithms
 
-## The *algorithm* argument
+This how-to guide is about the mechanics of specifying and configuring optimizers in
+optimagic. It is not about choosing the right algorithm for your problem. For a
+discussion on choosing algorithms, see
+[this how-to guide](how_to_algorithm_selection.ipynb)
 
-The `algorithm` argument can either be a string with the name of an algorithm that is
-implemented in optimagic, or a function that fulfills the interface laid out in
-{ref}`internal_optimizer_interface`.
+There are two ways to specify and configure optimizers. The *optimagic way* and the
+*scipy way*. Both use the `algorithm` argument of `minimize` and `maximize` to specify
+an optimizer and both are super easy to use.
 
-Which algorithms are available in optimagic depends on the packages a user has
-installed. We list all supported algorithms in {ref}`list_of_algorithms`.
+As the name suggests, the *scipy way* is more familiar for users of scipy.optimize. The
+*optimagic way* has much better discoverability and autocomplete. Using the *optimagic
```

Comment:
This sounds slightly negative toward scipy imo. I think you could rephrase like:

```suggestion
*optimagic way* adds discoverability and autocomplete. Using the *optimagic
```